pkg/jobs/trigger_at.go

```diff
@@ -0,0 +1,89 @@
+package jobs
+
+import (
+	"time"
+
+	"github.com/cozy/cozy-stack/web/jsonapi"
+)
+
+// AtTrigger implements the @at trigger type. It schedules a job at a specified
+// time in the future.
+type AtTrigger struct {
+	typ  string
```

Comment:
My question is: why do we need this field? I think that the `Type()` method can be implemented as `return a.in.Type`